nano/core_test/conflicts.cpp

```diff
@@ -44,7 +44,7 @@ TEST (conflicts, add_existing)
 	ASSERT_EQ (1, node1.active.size ());
 	{
 		std::lock_guard<std::mutex> guard (node1.active.mutex);
-		auto votes1 (node1.active.roots.find (nano::uint512_union (send2->previous (), send2->root ()))->election);
+		auto votes1 (node1.active.roots.find (nano::qualified_root (send2->previous (), send2->root ()))->election);
```

Comment:
We should add a function nano::block::qualified_root so these are always constructed correctly.

Reply:
Added function and updated applicable call sites